corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_properties.html

```diff
@@ -48,7 +48,8 @@ <h4 class="modal-title">
                         {% trans "Help Text" %}
                       </label>
                       <div class="col-xs-12 col-sm-9">
-                        <input class="form-control" type="text" data-bind="value: property.hint"/>
+                        <textarea class="form-control" type="text" data-bind="value: property.hint">
```

Comment:
Adding the `vertical-resize` class will let people stretch this box to be bigger.